### DISCUSSION TITLE
cmd filters

### DISCUSSION THREAD

**Discussion by haithium** [saulpw/visidata#2390] [Q&A]
Is there a way to open a sheet with filters from the command line? I'm trying to build a simple todo list sheet using VisiData, and I want to filter the status column to show only entries that are not marked as `Done`.

**Comment by midichef** [comment]
You can create a command file that does the filtering. Then, in the future, replay those commands by running `vd -p filter.vdj todo_list.ext`

To create the command file, run vd, carry out your filtering commands, then hit `Ctrl+D` (`save-cmdlog`) and save the command log to a file ending in `.vdj`.

**Reply by haithium** [reply]
Thanks for your help. Your comment was useful, but I don't like the idea of creating another file to filter the file.

**Comment by haithium** [comment]
Why isn't there a `--filter` flag or something that filters the sheets?

**Reply by anjakefala** [reply]
You can write a detailed feature request for the features you want: https://github.com/saulpw/visidata/issues/new?assignees=&labels=wishlist&projects=&template=feature-request.md&title=%5B%5D+ outling exactly how you would like it to work with a specific example. We will then consider it! The more detailed you make it, the more likely it will be picked up. If we have to guess how something will work, it is less likely to be implemented.

Do you mean "why doesn't the specific thing I want already exist in this project bursting with features and run entirely by volunteers" or do you mean "I'm curious if there is a technical or social reason the `--filter` flag does not already exist"?

**Reply by anjakefala** [reply]
Example, a type of filter that does exist, and the conversations that lead to it existing: https://github.com/saulpw/visidata/issues/1559
